tools/build.sbt

```diff
@@ -220,3 +229,55 @@ lazy val cloudflowOperator =
       dockerUsername := sys.props.get("docker.username"),
       dockerRepository := sys.props.get("docker.registry"),
       dockerBaseImage := "adoptopenjdk/openjdk11:alpine-jre")
+
+lazy val cloudflowDescriptorGenerator =
```

Comment:
Please rename to `cloudflowDescriptorExtractor` I think that is a better name. Use Extractor instead of Generatro everywhere?